app/assets/stylesheets/administrate/base/_forms.scss

```diff
@@ -79,6 +79,7 @@ textarea {
 }
 
 textarea {
+  min-height: 550px;
```

Comment:
Do you mean to change this in this PR too?

Reply:
oops, no I only meant to do that in my fork which I have an app using directly from github. I'll revert this change as soon as I can.